pkg/utils/config.go

```diff
@@ -459,6 +467,34 @@ func generateConfigFileFromTemplate(filePath string, templateString string, cont
 			return ip
 		case "registry.dist":
 			return defaults.DefaultRegistryDist
+
+		case "sdn.disable":
+			return false
```

Comment:
Let's make it optional (true by default) until we stabilize Eden tests workflow and merge.

Reply:
Done - disabled for now.